config/billing_rates.yml

```diff
@@ -10,3 +10,4 @@
 - { id: 80aa487a-0aa7-472f-a52c-d82a96067cba, resource_type: PostgresStorage, resource_family: standard, location: hetzner-fsn1,   unit_price: 0.000002443 }
 - { id: c41422cd-9e28-46bd-acd4-02debfcd26a2, resource_type: PostgresStorage, resource_family: standard, location: hetzner-hel1,   unit_price: 0.000002443 }
 - { id: c41422cd-9e28-46bd-acd4-02debfcd26a2, resource_type: PostgresStorage, resource_family: standard, location: github-runners, unit_price: 0.000002443 }
+- { id: d772d0aa-0b40-4b7a-aceb-72f6211f7cad, resource_type: GitHubRunner,    resource_family: standard, location: github-runners, unit_price: 0.0008 }
```

Comment:
One tricky thing is that there shouldn't be location of github runners. How about if we create billing record as `location: global` or `location: arbitrary` or `location: unknown`, then use that value while creating the billing record instead of vm location. Do you see any problems with that approach?

Reply:
I did that